backend/sample.env

```diff
@@ -0,0 +1 @@
+GOOGLE_PLACES_API_KEY=akjshdaksjhdjk
```

Comment:
This way we can monitor all our env variables without having to worry about accidentally commiting our `.env` file and everyone can be up to date with any environment variables they are missing. ReadME has some more instructions!